frontend/src/components/editor/output/MarimoErrorOutput.tsx

```diff
@@ -142,7 +142,7 @@ export const MarimoErrorOutput = ({
           </div>
         );
       case "strict-exception":
-        return error.blamed_cell == null ? (
+        return error.blamed_cell === null ? (
```

Comment:
`== null` can be nice since it covers `undefined` too. You may want to keep it that way.

Reply:
Agreed